Makd.mak

```diff
@@ -662,6 +662,13 @@ $O/doc.stamp: $(call find,$(SRC),-type f \( -name '*.d' -o -name '*.di' \))
 
 doc += $O/doc.stamp
 
+# General rule to run the harbored-mod generator
+$O/dfmt.stamp: $(shell find $(SRC) -type f \( -name '*.d' -o -name '*.di' \))
+	$(call exec,dfmt -i $(SRC) >/dev/null,fmt)
```

Comment:
From experience, it's better to have an easy way to override the `dfmt` binary to use, add a variable `DFMT ?= dfmt` as we have for `D1TO2FIX`, etc.

Also, isn't it possible to pass the files explicitly so there are no inconsistency between the dependencies and the files the command runs on? If so, please use `$(call exec,$(DFMT) -i $^ >/dev/null,dfmt,$(SRC))` or something like that.